src/Microsoft.SqlTools.ServiceLayer/Migration/MigrationService.cs

```diff
@@ -456,6 +571,38 @@ internal static List<AzureSqlSkuCategory> GetEligibleSkuCategories(string target
                                                     AzureSqlPaaSServiceTier.GeneralPurpose,
                                                     ComputeTier.Provisioned,
                                                     AzureSqlPaaSHardwareType.Gen5));
+                    //if (includePreviewSkus)
```

Comment:
why commented?

Reply:
The version of the NuGet I was using up until this point was the old one without the new SKUs so none of this actually worked. Now that we've released it I'll update it here and uncomment all this out